src/SFA.DAS.EAS.Portal.Client/Models/Concrete/AccountDto.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace SFA.DAS.EAS.Portal.Client.Models.Concrete
+{
+    public class AccountDto : IAccountDto
```

Comment:
No need for an interface on the ReadModel.   Add as tech debt in 454.
Should drop the Hungarian suffix.  The namespace gives the context.